as/plugins/org.jboss.ide.eclipse.as.core/jbosscore/org/jboss/ide/eclipse/as/core/util/ServerHomeValidationUtility.java

```diff
@@ -49,23 +48,27 @@ public IStatus validateServerHome(IServer server, boolean alwaysThrow) throws Co
 
 		IFilesystemController fs = (IFilesystemController)findDependencyFromBehavior(server, IFilesystemController.SYSTEM_ID);
 		IServerDetailsController det = (IServerDetailsController)findDependencyFromBehavior(server, IServerDetailsController.SYSTEM_ID);
-		IDeploymentOptionsController opts = (IDeploymentOptionsController)findDependencyFromBehavior(server, IDeploymentOptionsController.SYSTEM_ID);
-		IStatus s = validateServerHome(server, det, opts, fs);
+		IStatus s = validateServerHome(server, det, fs);
 		if( alwaysThrow && !s.isOK()) {
 			throw new CoreException(s);
 		}
 		return s;
 	}
 	
 	public IStatus validateServerHome(IServer server, IServerDetailsController details, 
-			IDeploymentOptionsController options, IFilesystemController fs) throws CoreException {
```

Comment:
You are breaking API of a public class/method here. Do we really have to do it? Why not deprecate the old method and add a new one. You can ignore options here.